src/backend/base/langflow/custom/custom_component/custom_component.py

```
@@ -425,8 +425,14 @@
         else:
             msg = f"Invalid user id: {self.user_id}"
             raise TypeError(msg)
-        async with session_scope() as session:
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)